tests/FakeItEasy.Specs/DummyCreationSpecs.cs

```diff
@@ -510,6 +510,22 @@ public void LazyCreationNonDummyable(Lazy<NonDummyable> dummy)
                 .x(() => dummy.Value.Should().BeNull());
         }
 
+        [Scenario]
+        public void LazyCreationIsLazy(Lazy<LazilyCreated> dummy)
+        {
+            "Given a type that tracks instance creation"
+                .x(() => LazilyCreated.IsInstanceCreated = false);
+
+            "When a dummy lazy of this type is requested"
+                .x(() => dummy = A.Dummy<Lazy<LazilyCreated>>());

```

Comment:
I see you took a page out of my book and spelled `this.CreateDummy` funny.

Reply:
Oops!